SpatialGDK/Source/SpatialGDKEditorCommandlet/Private/Commandlets/CookAndGenerateSchemaCommandlet.cpp

```diff
@@ -161,3 +177,234 @@ int32 UCookAndGenerateSchemaCommandlet::Main(const FString& CmdLineParams)
 
 	return CookResult;
 }
+
+int32 UCookAndGenerateSchemaCommandlet::MainCook(const FString& CmdLineParams, FObjectListener& Listener)
+{
+	if (0)
+	{
+		return Super::Main(CmdLineParams);
+	}
+	else
+	{
+		// Copied from UCookCommandlet::CookByTheBook to fill cook parameters
```

Comment:
Is all the code below copied? or are there some differences?

Reply:
I just skip some parameters, but no difference. We could avoid copying it if we refactored the engine cook command.

Reply:
The code I added starts line 345